module/components/Avatar/Readme.md

```diff
@@ -1,36 +1,14 @@
 Avatar:
 
 ```js
-<Avatar backgroundColor="#333" size="100"> </Avatar>
-
-```
-
-Image Avatar:
-```js
-
-```
-
-Image Avatar with custom size:
- ```js
- 
- ```
-
-FontIcon Avatar:
-```js
-
-```
-
-FontIcon Avatar with custom colors and size:
+<Avatar color="info" src='styleguide/avatar_girl.jpg' />
+  ```
 ```js
+<Avatar color="info" src='styleguide/avatar_boy.jpg' />
```

Comment:
Please change `*_boy` to something neutral.

Reply:
Fixed